test/CadlRanchProjects.Tests/type-array.cs

```diff
@@ -220,5 +220,13 @@ public Task Type_Array_NullableModelValue_put() => Test(async (host) =>
             var response = await new ArrayClient(host, null).GetNullableModelValueClient().PutAsync(new List<InnerModel> { new InnerModel("hello"), null, new InnerModel("world") });
             Assert.AreEqual(204, response.Status);
         });
+
+        [Test]
+        public void NotRequiredNullableListsAreNotNullByDefault()
```

Comment:
I think we should use `Optional`.
Negative phrases might be misleading

Reply:
fixed